system/cfme-setup.sh

```diff
@@ -31,6 +31,17 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+# Incorporate the iptable updates
+if [[ -f /etc/sysconfig/iptables.miq ]]; then
+  /sbin/iptables-save > /etc/sysconfig/iptables.SYSTEM
+  /sbin/iptables-restore /etc/sysconfig/iptables.miq
+  /sbin/service iptables save
+  # Expect no more than the four datestamp differences.
+  if [[ $(/usr/bin/diff -Ew -U 0 /etc/sysconfig/iptables /etc/sysconfig/iptables.miq  | /bin/grep "^@@" | /usr/bin/wc -l) -gt 4 ]]; then
```

Comment:
@Fryguy I had added the conditional based on a conversation with @jrafanie, where he had initially suggested attempting to diff what is in the kernel tables with the iptables.miq we seeded with to confirm we seeded correctly. The 4 diffs result from the how /sbin/iptables-restore adds 2 lines of header and 2 lines of trailer to the iptables file it generates.
Although having this conditional could help identify potential build failures
where future changes to iptables.miq failed to apply, if it seems confusing to
folks I can remove it.
Thoughts?